src/LibVLCSharp/Shared/Helpers/MarshalUtils.cs

```diff
@@ -509,7 +513,7 @@ internal static T PtrToStructure<T>(IntPtr ptr)
 #if NETSTANDARD1_1 || NET40
             return (T)Marshal.PtrToStructure(ptr, typeof(T));
 #else
-            return Marshal.PtrToStructure<T>(ptr);
+            return Marshal.PtrToStructure<T>(ptr)!;
```

Comment:
I have a doubt here, but I think that `T` should be written as `T?` on the caller side to allow it to return a null, what do you think ?
Seems like allowing null here would break 20 usage locations.

Reply:
`PtrToStructure` would return an empty struct but not null, right?

Reply:
T might be a ref type, I guess. the compiler was yelling anyway.
Maybe we could add a `struct` constraint

Reply:
> T might be a ref type, I guess.

Could but we don't marshal ref types currently. No strong feelings tho

Reply:
Then I'd be in favor of adding a struct constraint to the method and remove the assertion here

Reply:
Sure